topiary/src/atom_collection.rs

```diff
@@ -120,6 +120,12 @@ impl AtomCollection {
             log::debug!("Aborting because context is single-line and #multi_line_only! is set");
             return Ok(());
         }
+        if let Some(parent_id) = self.parent_leaf_nodes.get(&node.id()) {
+            if *parent_id != node.id() {
+                log::warn!("Aborting because the match occurred below a leaf node: {node:?}");
```

Comment:
This also applies to the other warnings above this one, but "abort" is a bit strong. If I saw that in the logs, I might think that the whole program had halted, but that's not the case. Maybe "skip" would be less alarmist.

Reply:
Replaced with `Skipping` in the latest version